kubernetes/samples/charts/util/setupLoadBalancer.sh

```diff
@@ -13,6 +13,12 @@ UTILDIR="$(dirname "$(readlink -f "$0")")"
 #Kubernetes command line interface. 
 #Default is 'kubectl' if KUBERNETES_CLI env variable is not set.  
 kubernetesCli=${KUBERNETES_CLI:-kubectl}
+okecluster=${OKE_CLUSTER:-false}
```

Comment:
This is a sample script, when you change the script we need to update the document and usage 

Need to document the usage of new env variable OKE_CLUSTER. Can kubernate auto detect OKE environment ?    

Why the policy need to to Always in case of OKE ? If we keep IfNotPresent the policy it pulls the image if not present. Then why to put "Always" explicitly.

Reply:
added comment , in OKE cluster with multiple nodes it is recommended to use Always to avoid intermittent failures

Reply:
Is this a BUG in OKE environment ? If we push the images to all nodes  IfNotPresent poicly should work.